samples/meeting-tabs/nodejs/teamsAppManifest/manifest.json

```diff
@@ -0,0 +1,69 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.13",
+  "version": "1.0.0",
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.teams.meetingtabs",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "color": "color.png",
+    "outline": "outline.png"
+  },
+  "name": {
+    "short": "Meeting-Tabs-Nodejs",
+    "full": "Meeting-Tabs-Nodejs"
+  },
+  "description": {
+    "short": "Meeting-Tabs",
+    "full": "This sample shows app stage view, Mute/Unmute Teams meeting audio call and Side panel."
```

Comment:
This sample shows app stage view, Mute/Unmute Teams meeting audio call in meeting Side panel tab.

Reply:
Fixed